### PR TITLE
[DOC UPDATE] Translate Tutorial to Chinese 10/15

### PR DESCRIPTION
**Pull request by kingswanwho**
# [DOC UPDATE] Translate Tutorial to Chinese 10/15

## Description

Translate a tutorial to Chinese:

010-installing-the-apache-drill-sandbox.md

Overall 10 tutorials have been translated, 5 tutorials to go.

## Documentation

Please refer to Chinese tutorial.

## Testing

This is a doc update, no test here.

### CODE REVIEW

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md [hunk 1]
```diff
@@ -58,66 +54,64 @@ VMware Player or VMware Fusion:
               Log in to this virtual machine: Linux/Windows <Alt+F2>, Mac OS X <Options+F5>
 **Note:** The URL provided corresponds to the Web UI in Apache Drill.
```
Comment (luocooong) [inline _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md:55]: Not translated. Keep consistent with the next NOTE text.

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md [hunk 2]
```diff
@@ -58,66 +54,64 @@ VMware Player or VMware Fusion:
...
               You can access the host via SSH by ssh mapr@localhost -p 2222
               Log in to this virtual machine: Linux/Windows <Alt+F2>, Mac OS X <Options+F5>  
 
-       **Note:** The URL provided corresponds to the Web UI in Apache Drill.        
+       **注意：** 提供的 URL 对应于 Apache Drill 中的 Web UI。      
 
-10. Navigate to [localhost:8047](http://localhost:8047) to experience the Drill Web UI, or log into the sandbox through the command line.  
-       * Login using ssh as described in ["Getting to Know the Sandbox"]({{site.baseurl}}/docs/getting-to-know-the-drill-sandbox). When prompted, enter `mapr` as the login name and password.  
-       * Alternatively, access the command line on the VM: Press Alt+F2 on Windows or Option+F5 on Mac.   
+   1. 切换到 [localhost:8047](http://localhost:8047) 来体验 Drill web UI，或者通过命令行登录 MapR Sandbox。
```
Comment (luocooong) [inline _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md:110]: Should be sequence 10.

**Review by luocooong — CHANGES_REQUESTED** [review]
@kingswanwho Thanks for the contribution. Please note the few comments. Then, check the full-width characters (period) at the end of each sentence.

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md (hunk 1 above)
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md]: Actually, I found that the "Note" should be part of MapR installation message.
So I keep both of the two "Note" in English.

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md (hunk 2 above)
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/010-installing-the-apache-drill-sandbox.md]: Solved

**Review by luocooong — APPROVED** [review]
LGTM. +1

**Pull request merged**